aiohasupervisor/root.py

```diff
@@ -117,11 +117,14 @@ async def reload_updates(self) -> None:
         await self._client.post("reload_updates", timeout=ClientTimeout(total=300))
 
     async def refresh_updates(self) -> None:
-        """Refresh updates (discouraged)."""
+        """Refresh updates.
+
+        Discouraged. Use the `reload_updates()` and `store.reload()` instead.
```

Comment:
I'd say something about why it is discouraged.

Reply:
Updated. Not sure if that is a good reason really, but it is why I plan to not use the endpoint from Core side anymore. With that, this endpoint likely has no users at all, so I'd rather prefer to remove it at one point.